Chinese_Remainder_Theorem/Chinese_Remainder_Theorem.js

```diff
@@ -0,0 +1,59 @@
+/*
+The Chinese Remainder Theorem or CRT as it is commonly known is used to evaluate the smallest number 'n' such that:
+n % numbers[0] = remainder[0],
+n % numbers[1] = remainder[1],
+..............................
+n % numbers[length-1] = remainder[length-1].
+CRT belongs to the class of residue number system.
+CRT is given by relation n=Σ{s(i)*(remainders(i)*s1_inverse(i))mod divisors(i)}mod M, where:
+1. s(i) is partial product.
+2. s(i)_inverse is the inverse for partial product.
+3. remianders(i) is the remiander set value.
+4. divisors(i) is the divisor set value.
+5. M is known as the dynamic range
+*/
+
+//the inverse element is defined by (s1*s1_inverse)%moduli =1, where s1_inverse needs to be evaluated.
+const inverseElement = (s1, moduli) => {
+    let x_primary = 0;
+    let x_secondary = 1;
+    let moduli_primary = moduli;
```

Comment:
keep a blank line after declaring variable

Reply:
I replaced this mentioned with a more simpler method. I hope that's okay now